setup.py

```diff
@@ -35,5 +35,8 @@
             'console_scripts': [
                 'sep = sideboard.sep:run_plugin_entry_point'
             ]
+        },
+        extras_require={
+            'perftrace': ['python-prctl>=1.6.1', 'psutil>=4.3.0']
```

Comment:
See https://github.com/magfest/sideboard/issues/25 for the discussion of why we moved these dependencies into ``extras_require``.

This WILL require a small MAGFest puppet change!  Otherwise we won't install these dependencies, since we're not running ``pip install -e .[perftrace]`` when installing dependencies.